test/fixtures/test_grammars/depends_on_column/grammar.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "depends_on_column",
```

Comment:
These test grammars can now be written in JS instead of JSON, and I've been gradually converting them all to JS, since it's so much more readable. Since this grammar is a new addition, would you mind converting it to JS?

Reply:
> These test grammars can now be written in JS instead of JSON, and I've been gradually converting them all to JS, since it's so much more readable. Since this grammar is a new addition, would you mind converting it to JS?

np, converted to grammar.js